src/components/datepicker/utils.ts

```diff
@@ -18,3 +19,53 @@ export const isSelectingFirstDay = (from: Date, to: Date, day: Date) => {
   const isRangeSelected = from && to;
   return !from || isBeforeFirstDay || isRangeSelected;
 };
+
+export const isAllowedDate = (date: Date, disabledDays?: AllowedDisabledDays) => {
+  if (disabledDays) {
+    date.setHours(0, 0, 0, 0);
```

Comment:
Looks like a utility function like
```
const getDay = (date: DateTime) => date.setHours(0, 0, 0, 0);
```

Could simplify this code a bunch (it also already returns a timestamp so no need to use `getTime` either)

\* Naming could be better 😅